src/application/Application.mjs

```diff
@@ -405,6 +421,195 @@ export default class Application extends Component {
         return;
     }
 
+    _recieveScrollWheel(e) {
+        const obj = e;
+        const {clientX, clientY, deltaY} = obj;
+
+        if (clientX <= this.stage.w && clientY <= this.stage.h) {
+            if (deltaY > 0) {
+                if (!this.fireTopDownScrollWheelHandler("_captureScrollDown")) {
+                    this.fireBottomUpScrollWheelHandler("_handleScrollDown", obj);
+                }
+            } else if (deltaY < 0) {
+                if (!this.fireTopDownScrollWheelHandler("_captureScrollUp")) {
+                    this.fireBottomUpScrollWheelHandler("_handleScrollUp", obj);
+                }
+            }
+        }
+    }
+
+    fireTopDownScrollWheelHandler(event) {
+        let children = this.stage.application.children;
+        let affected = this._findChildren([], children).reverse();
+        let n = affected.length;
+
+        while(n--) {
+            const child = affected[n];
+            if (child && child[event]) {
+                if (event === "_captureScrollDown") {
+                    child._captureScrollDown();
+                    return true;
+                } else if (event === "_captureScrollUp") {
+                    child._captureScrollUp();
+                    return true;;
+                }         
+            }
+        }
+        return false
+    }
+
+    fireBottomUpScrollWheelHandler(event, obj) {
```

Comment:
Scroll handler should pass the event to the hook so that the developer can access its deltaX and deltaY values. As a consequence, there is no need for up/down distinction, just `handleScroll`.